app/javascript/components/common/svg-graph-util.ts

```diff
@@ -7,119 +7,221 @@ export type Line = {
   length: number
   angle: number
 }
-
-const SMOOTHING_RATIO = 0.2
+const SIMILAR_THRESHOLD_DELTA = 0.01
```

Comment:
if points are within 1% of each other of each other, they will be considered equal for the purposes of smoothing.